apps/server/app/api/gateway/usage/route.ts

```
@@ -0,0 +1,74 @@
+  }
+
+  let plan: Plan | null;
+  try {
+    plan = await readEntitlement({ polar, products: process.env }, identity);
+  } catch (error) {
+    console.error("readEntitlement failed:", error);
+    return Response.json({ code: "entitlement_error" }, { status: 503 });
+  }
+
+  const window = windowForPlan(plan, now);
+  let rows;
```

Biome (2.5.7)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)